StudentManagement/StudentManagement.Console/Program.cs

```diff
@@ -61,8 +90,23 @@
 
             }
 
+
+            break;
+        case 3:
+            if (!File.Exists(filePath))
+            {
+                Console.WriteLine("Creating a new json file...");
+                File.Create(filePath).Close();
+
+            }
             break;
+        case 4:
 
+            File.WriteAllText(filePath, studentJson, Encoding.UTF8);
+            string readText = File.ReadAllText(filePath);
```

Comment:
what is the point of deserialization here?

Reply:
Deserialization is now when i check if json file exists, then i read all text from file and deserializing object